woocommerce/woocommerce-android
# issues/874-email-from-order-notif

Fixes #874 - the problem was due to `OrderDetailCustomerInfoView` relying on an `OrderCustomerActionListener` to do the work of contacting the customer, and that listener only existed if an order was shown from the order list.

This PR changes it so `OrderDetailCustomerInfoView` uses a new `OrderCustomerHelper` to contact the customer, which means it all works regardless of where the detail was shown.

To test:

- Open an order from the order list
- Verify that you can still email, text, or call the customer
- Open an order from the notifs list
- Verify that you can now email, text, or call the customer

Update release notes:
